### ISSUE TITLE
config.globalStyle scss @imports are not watched

### ISSUE THREAD

**Issue by AlexSapoznikov** [ionic-team/stencil#2741]
**Stencil version:**

```
 @stencil/core@2.3.0
```

**I'm submitting a:**

[x] bug report
[ ] feature request
[ ] support request => Please do not submit support requests here, use one of these channels: https://stencil-worldwide.herokuapp.com/ or https://forum.ionicframework.com/

**Current behavior:**
**stencil.config.ts** defines globalStyle like this:

```
import { sass } from '@stencil/sass';

export const config: Config = {
  namespace: 'wapi-ui',
  outputTargets: [
    {
      type: 'dist',
      esmLoaderPath: '../loader',
    },
    {
      type: 'dist-custom-elements-bundle',
    },
    {
      type: 'docs-readme',
    },
    {
      type: 'www',
      serviceWorker: null, // disable service workers,
    },
  ],
  globalStyle: 'src/global.scss',
  plugins:[

    sass()
  ],
};
```

and global.scss looks like this:

```
// Import all scss here

@import "styles/web";
@import "styles/icons";
```

The issue is that any change in those files that are imported do not trigger rebuild in dev mode.

**Expected behavior:**

Changes in global style imports should be watched as well and trigger rebuild in development mode. 

**Steps to reproduce:**
Implement the code above and try changing scss in imported file.

**Comment by claviska** [comment]
I've noticed this many times before and can confirm. Styles that get imported from the global stylesheet are not watched.

@adamdbradley is this something we can solve via config in @stencil/sass or does it have to be addressed directly in Stencil?

**Comment by AlexSapoznikov** [comment]
@claviska
Is it something that could be solved in the near future? It is quite frustrative to rebuild manually all the time.. :)

**Comment by christian-bromann** [comment]
It seems like global styles aren't working at all. I created a reproducible example with latest Stencil version: https://github.com/christian-bromann/stencil-global-styles-repro. Will ingest into our backlog.

**Comment by christian-bromann** [comment]
> It seems like global styles aren't working at all.

I forgot to add a style sheet to my template as mentioned in the Stencil docs:

> This file must be manually imported in the index.html of your application.
> 
> ```ts
> <link rel="stylesheet" href="/build/app.css" />
> ```

**Comment by christian-bromann** [comment]
This is a duplicate of #1908 where Stencil fails to update changes within nested css or sass files. The issue has been addressed in #5244

**Comment by christian-bromann** [comment]
The fix for this issue has been released as a part of today's [Stencil v4.11.0 release](https://github.com/ionic-team/stencil/releases/tag/v4.11.0).
